e2e/setup/config.go

```diff
@@ -207,14 +208,14 @@ func newSetup(config Config) (*Setup, error) {
 // clients used by the e2e tests
 type clients struct {
 	Hedera          *hederaSDK.Client
-	EVM             map[uint64]EVMUtils
+	EVM             map[uint64]evmSetup.Utils
 	MirrorNode      *mirror_node.Client
 	ValidatorClient *e2eClients.Validator
 	FeeCalculator   service.Fee
 	Distributor     service.Distributor
 }
 
-func routerAndEVMTokenClientsFromEVMUtils(evmUtils map[uint64]EVMUtils) (
+func routerAndEVMTokenClientsFromEVMUtils(evmUtils map[uint64]evmSetup.Utils) (
```

Comment:
Could you please pick one of the two `RouterAndEVMTokenClientsFromEVMUtils` between `e2e/setup/config.go` and  `e2e/setup/evm/setup.go`?